server.js

```diff
@@ -4,6 +4,7 @@ import passport from 'passport';
 import passport_facebook from 'passport-facebook';
 import session from 'express-session';
 import cookieParser from 'cookie-parser';
+import bodyParser from 'body-parser';
```

Comment:
Oh we actually dont need the body parser i think, that's left over from my debugging. I'll remove it.